owslib/wps.py

```diff
@@ -863,34 +863,36 @@ def getOutput(self, filepath=None):
         if self.isSucceded():
             content = b''
             for output in self.processOutputs:
-
-                output_content = output.retrieveData(
-                    self.auth.username, self.auth.password,
-                    headers=self.headers, verify=self.auth.verify, cert=self.auth.cert)
+                if identifier and output.identifier != identifier:
```

Comment:
For clarity, set identifier explicitly to the first output. 
Use filter instead of continue. 
Remove break

Reply:
done.